agent/src/beerocks/monitor/CMakeLists.txt

```diff
@@ -1,8 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
-
-set(PROJECT_NAME beerocks_monitor)
-project (${PROJECT_NAME})
-message("${BoldWhite}Preparing ${BoldGreen}${PROJECT_NAME}${BoldWhite} for the ${BoldGreen}${TARGET_PLATFORM}${BoldWhite} platform${ColourReset}")
```

Comment:
I disagree that it's just noise - it helps developers spot the areas in the configuring stage. I prefer to keep it, but I'd like to hear what @vitalybu thinks.

Reply:
I agree with @tomereli.
prplMesh contains several components and it does help to see the progress of the configuration stage.

Reply:
OK I'll keep the message calls.

It was more of an accident - I had removed it in 3 files and then I thought I can just as well continue doing it :-)